clearwater-infrastructure/usr/share/clearwater/bin/set_ntp_ip

```diff
@@ -0,0 +1,67 @@
+#Copyright (C) 2016 Metaswitch Networks Ltd
+
+import os, pipes, platform, subprocess, sys
+from datetime import datetime
+
+NTP_CONFIG="/etc/ntp.conf"
+KEY="server"
+SCRIPT="/usr/share/clearwater/bin/set_ntp_ip"
```

Comment:
You might just be able to use `sys.argv[0]` or `__file__` here to get the script's own name.